server/server.go

```diff
@@ -212,7 +212,12 @@ func (s *Server) restore(ctx context.Context) []string {
 			pods[containers[i].ID] = &metadata
 		} else {
 			podContainers[containers[i].ID] = &metadata
-			containersAndTheirImages[containers[i].ID] = containers[i].ImageID
+			imageID, err := storage.ParseStorageImageIDFromOutOfProcessData(containers[i].ImageID)
+			if err != nil {
+				log.Warnf(ctx, "Error parsing image ID %q of container %q: %v, ignoring", containers[i].ImageID, containers[i].ID, err)
+				continue
+			}
+			containersAndTheirImages[containers[i].ID] = imageID
```

Comment:
Looking at coverage data… apparently the existing `"should succeed with container restore"` test exercises this using 3 containers with `ID` and `ImageID` both empty. I’m… getting tired of fixing mocks that just don’t represent real-world operation.

Reply:
I have changed that to at least use 3 proper, and distinct, values. Not that it exercises the wipe logic all that much…